### ISSUE TITLE
estadiofutebol.com

### ISSUE THREAD

**Issue by X01X012013** [X01X012013/AdBlockProtector#118]
http://www.estadiofutebol.com/gratis/motorstv

https://github.com/reek/anti-adblock-killer/issues/3030

**Comment by X01X012013** [comment]
Solution: 
```
@@||cdn.tvdez.com^$domain=www.estadiofutebol.com
||cdn.tvdez.com^$subdocument,domain=www.televisaofutebol.com
```

**Comment by X01X012013** [comment]
New solution: `@@||cdn.tvdez.com^$image,domain=www.estadiofutebol.com`

Doesn't always work, seems to be related to cookies.

**Comment by X01X012013** [comment]
The link-icons are fixed, the iframe will be handled in #116 .
